platform/android/display_server_android.cpp

```diff
@@ -216,6 +216,14 @@ DisplayServer::ScreenOrientation DisplayServerAndroid::screen_get_orientation(in
 	return (ScreenOrientation)orientation;
 }
 
+int DisplayServerAndroid::screen_get_internal_current_rotation(int p_screen) const {
+	GodotIOJavaWrapper *godot_io_java = OS_Android::get_singleton()->get_godot_io_java();
+	ERR_FAIL_NULL_V(godot_io_java, SCREEN_LANDSCAPE);
```

Comment:
This returns `SCREEN_LANDSCAPE` on failure, but the values returned by the java implementation are `0, 90, 180, 270` which don't map to the `ScreenOrientation` enum.

Reply:
Good point. SCREEN_LANDSCAPE mapped to 0 which was the intended value.
Anyway, changed to 0.